spec/components/sdg/goals/show_component_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe SDG::Goals::ShowComponent, type: :component do
+  let!(:goal_1) { SDG::Goal[1] }
+
+  before do
+    Setting["feature.sdg"] = true
+  end
+
+  it "renders a heading" do
+    component = SDG::Goals::ShowComponent.new(goal_1)
+
+    render_inline component
+
+    expect(page).to have_css ".goal-title"
+    expect(page).to have_content "No Poverty"
+  end
+
+  it "renders a long description" do
+    component = SDG::Goals::ShowComponent.new(goal_1)
+
+    render_inline component
+
+    expect(page).to have_css "#description_goal_#{goal_1.code}"
+    expect(page).to have_content "Globally, the number of people living in extreme poverty declined from 36 per cent in 1990 to 10 "
```

Comment:
Maybe a bit less information is enough :smiling_imp:.

```suggestion
    expect(page).to have_content "Globally, the number of people living in extreme poverty"
```